src/language.cpp

```diff
@@ -69,6 +69,8 @@
 
 Translator *theTranslator=0;
 
+void config_warn(const char *fmt, ...);
```

Comment:
a bit tricky to depend on internal functions of `config_impl.l`. Maybe better to add a `err_non_fatal`? (like `err_full` is now, but maybe that should have had a `handle_warn_as_error()` as well?)

Reply:
It is indeed a bit tricky.
I thought about something like `err_non_fatal` as well but I decided to do it this way as I wanted the message to have the config look and feel as well as the place (so not into the warnings log file). Maybe a solution would be to have a separate include file or something like it.